src/commcare_cloud/ansible/ansible.cfg

```diff
@@ -11,5 +11,10 @@ fact_caching_timeout = 21600
 
 timeout = 30
 
+inventory_plugins = ./plugins/inventory
```

Comment:
For me this only works if i'm already in the `src/commcare_cloud/ansible` directory. I was trying to run `cchq icds-cas x` but this plugin would not be found

Reply:
odd, it should be relative to this file (same as callback plugins)

Reply:
I this should fix it: https://github.com/dimagi/commcare-cloud/pull/2050/files#diff-9af49c7720ecc05ddbc9456b1c90b73bR10